api/services/handle_event_service.py

```diff
@@ -9,7 +9,9 @@
 
 class HandleEventService:
     def handle_event_service(self, form_data):
-        call_sid_exist = models.CallLogEvent.query.call_sid_exist(form_data)
+        call_sid_exist = models.CallLogEvent.query.call_sid_exist(
+            form_data.get("CallSid")
+        )

```

Comment:
Previously, the entire webhook payload was passed as a parameter in the query, and we extracted the `callsid` within the model. Now, I’ve optimized the process by passing only the `callsid` in the query, as it's the only required parameter.